js/modules/k6/grpc/client_test.go

```diff
@@ -261,6 +262,123 @@ func TestClient(t *testing.T) {
 				client.load([], "../../../../lib/testutils/httpmultibin/grpc_testing/test.proto");`},
 			vuString: codeBlock{code: `client.connect("GRPCBIN_ADDR", { timeout: 3456.3 });`},
 		},
+		{
```

Comment:
None of this test seem to test that we do *actually* use the configured tlsconfig to do the connection and error out if we use the wrong certificate

Reply:
Solid point. I'll sort this out.

Reply:
Solid point. Working on this now.